WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -206,9 +209,12 @@ final class AuthenticationManagerTests: XCTestCase {
         XCTAssertTrue(rootController is ULAccountMismatchViewController)
     }
 
-    func test_it_can_display_jetpack_error_for_org_site_credentials_sign_in() {
+    func test_it_can_display_jetpack_error_for_org_site_credentials_sign_in_when_not_using_application_password_authentication() {
```

Comment:
Nit: similar to the above comment, should we add a test case for when the treatment variant is assigned?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8769/commits/d224425421c86a5bb0c83c794592ef3b89aaf9c0